CSS/main.css

```diff
@@ -605,19 +659,29 @@ body .featured .seeMore .down {
   width: 15px;
   height: 15px;
 }
+body #article {
+  display: none;
+}
 body footer {
   display: flex;
   align-items: flex-start;
   flex-direction: column;
+  display: none;
   position: relative;
   top: 20px;
   width: 375px;
 }
+@media (min-width: 400px) and (max-width: 767px) {
+  body footer {
+    width: 100%;
+  }
+}
 @media (min-width: 768px) {
   body footer {
     height: 300px;
     width: 100%;
     top: 100px;
+    display: flex;
```

Comment:
- You have done a great job implementing this website, it is great on the mobile version(375px) and screen size of `1440px` .  But on screen size `768px` and `1024px`, the content of your website seem squashed up, and a bit distorted. 
- It is important for our website to responsive on different screensizes and that's one of the requirements of this project. Can we fix this? 🤗 

Here is what i can see 👇🏽 

Here is what I can see compared to what's on Figma 👇🏽 
| On Screensize 768px  |
| :---:   | 
|<img width="588" alt="Screenshot 2023-02-16 at 16 40 03" src="https://user-images.githubusercontent.com/99246359/219505364-44daaca0-75f0-45c8-a816-915fef142aa0.png">
  |

| On Screensize 768px  |
| :---:   | 
|<img width="586" alt="Screenshot 2023-02-16 at 16 40 25" src="https://user-images.githubusercontent.com/99246359/219505461-002b2a99-48c3-41a1-aaa6-ef94692e7bd1.png">
  |

| On Screensize 768px  |
| :---:   | 
|<img width="595" alt="Screenshot 2023-02-16 at 16 40 55" src="https://user-images.githubusercontent.com/99246359/219505561-fce313ba-1c5a-4f34-b206-8152e26ceb56.png">
  |


| On Screensize 768px |
| :---:   | 
|<img width="596" alt="Screenshot 2023-02-16 at 16 41 18" src="https://user-images.githubusercontent.com/99246359/219505642-29b45ea4-53ed-41c8-b114-da5965ea77bc.png">
  |


| On Screensize 768px  |
| :---:   | 
|<img width="582" alt="Screenshot 2023-02-16 at 16 41 41" src="https://user-images.githubusercontent.com/99246359/219505736-989836d1-bd96-415a-aea7-0d0226cd2b44.png">
  |


| On Screensize 1024px  |
| :---:   | 
|<img width="776" alt="Screenshot 2023-02-16 at 17 03 18" src="https://user-images.githubusercontent.com/99246359/219508221-33778a76-f98d-4637-82eb-8e1f33b34c16.png">
  |

| On Screensize 1024px  |
| :---:   | 
|<img width="771" alt="Screenshot 2023-02-16 at 17 03 54" src="https://user-images.githubusercontent.com/99246359/219508240-47aaf82a-75ad-435e-b980-79c3d07506de.png">
  |